app/kumactl/cmd/generate/generate_dataplane_token_test.go

```diff
@@ -100,4 +100,29 @@ var _ = Describe("kumactl generate dataplane-token", func() {
 		// and
 		Expect(buf.String()).To(Equal("Error: failed to generate a dataplane token: could not connect to API\n"))
 	})
+
+	It("should throw an error when dataplane token server is disabled", func() {
+		// setup
+		ctx.Runtime.NewCatalogueClient = func(s string) (catalogue_client.CatalogueClient, error) {
+			return &test_catalogue.StaticCatalogueClient{
+				Resp: catalogue.Catalogue{
+					Apis: catalogue.Apis{
+						DataplaneToken: catalogue.DataplaneTokenApi{
+							LocalUrl: "", // disabled dataplane token server
+						},
+					},
+				},
+			}, nil
+		}
+
+		// when
+		rootCmd.SetArgs([]string{"generate", "dataplane-token", "--dataplane=example"})
+		err := rootCmd.Execute()
+
+		// then
+		Expect(err).To(HaveOccurred())
+
+		// and
+		Expect(buf.String()).To(Equal("Error: failed to create dataplane token client: Dataplane Token Server in Kuma CP is disabled. Either enable the server or run Kuma DP without token.\n"))
```

Comment:
> or run Kuma DP without token

Why is this message about `Kuma DP` ? It's `app/kumactl/cmd/generate/generate_dataplane_token_test.go`